src/Windows/ArcGISRuntime.Windows.Samples/Samples/MapView/DisplayLayerViewState/DisplayLayerViewStateVB.xaml

```diff
@@ -14,16 +14,10 @@
             <RowDefinition Height="auto" />
         </Grid.RowDefinitions>
         <esriUI:MapView x:Name="MyMapView"/>
-        <ListView x:Name="layerStatusListView" Grid.Row="1">
-            <ListView.ItemTemplate>
-                <DataTemplate>
-                    <TextBlock Margin="2">
-                        <Run Text="{Binding LayerName, Mode=OneWay}"/>
-                        <Run Text=" - " />
-                        <Run Text="{Binding LayerViewStatus, Mode=OneWay}"/>
-                    </TextBlock>
-                </DataTemplate>
-            </ListView.ItemTemplate>
-        </ListView>
+      <StackPanel Orientation="Vertical">
```

Comment:
Add border (with white background) to make the text more readable (especially if it appears over the "for beta use only" watermark).
```
<Border
    Background="White" BorderBrush="Black" BorderThickness="1"
    HorizontalAlignment="Left" VerticalAlignment="Top"
    Margin="30" Padding="20" Width="250">
    <StackPanel Orientation="Vertical">
        <TextBlock x:Name="StatusLabel_TiledLayer" Text=""/>
        <TextBlock x:Name="StatusLabel_ImageLayer" Text=""/>
        <TextBlock x:Name="StatusLabel_FeatureLayer" Text=""/>
    </StackPanel>
</Border>
```